From: Markus Lehtonen <markus.lehtonen@linux.intel.com>
Date: Tue, 15 Jan 2013 14:37:39 +0200
Subject: [PATCH] packaging: change my2.patch to git email format

Change my2.patch from plain diff to git email format to test author and
date parsing of gbp.

Signed-off-by: Markus Lehtonen <markus.lehtonen@linux.intel.com>

diff --git a/packaging/my2.patch b/packaging/my2.patch
index ad5ca2d..5554d9c 100644
--- a/packaging/my2.patch
+++ b/packaging/my2.patch
@@ -1,3 +1,12 @@
+From fb613432f15bc199a724741883b23fff9946b567 Mon Sep 17 00:00:00 2001
+From: Markus Lehtonen <markus.lehtonen@linux.intel.com>
+Date: Tue, 13 Nov 2012 13:55:46 +0200
+Subject: [PATCH] My modification
+
+Add new file 'mydir/myfile.txt'.
+
+Signed-off-by: Markus Lehtonen <markus.lehtonen@linux.intel.com>
+
 diff --git a/mydir/myfile.txt b/mydir/myfile.txt
 new file mode 100644
 index 0000000..2cdad29
